App.js

```diff
@@ -19,14 +18,7 @@ import AddSiteScreen from './src/screens/AddSiteScreen';
 import MoreScreen from './src/screens/MoreScreen';
 import AuthScreen from "./src/screens/AuthScreen";
 
-const store = createStore(
-    reducers,
-    {},
-    compose(
-        applyMiddleware(ReduxThunk)
-    ));
```

Comment:
I move the creation of the Store to its own file, since it got pretty complex with all the `redux-persist` work.